app/javascript/src/components/Dashboard/Contacts/Pane/Form.jsx

```diff
@@ -0,0 +1,94 @@
+import React, { useState } from "react";
+
+import { Formik, Form } from "formik";
+import { Button, Pane, Toastr } from "neetoui";
+import { Input, Select } from "neetoui/formik";
+
+import {
+  CONTACTS_FORM_VALIDATION_SCHEMA,
+  ROLES,
+  PROFILE_PICTURE_URL,
+} from "../constants";
+
+const ContactForm = ({ onClose, contact, setContacts }) => {
+  const [submitted, setSubmitted] = useState(false);
+
+  const handleSubmit = values => {
+    setContacts(contacts => [
+      {
+        ...values,
+        id: contacts.length + 1,
+        role: values.role.label,
+        imageUrl: PROFILE_PICTURE_URL,
+        createdAt: new Date(),
+      },
+      ...contacts,
+    ]);
+    Toastr.success("Contact has  successfully created");
+    onClose();
+  };
+
+  return (
+    <Formik
+      initialValues={contact}
+      validateOnBlur={submitted}
+      validateOnChange={submitted}
+      validationSchema={CONTACTS_FORM_VALIDATION_SCHEMA}
+      onSubmit={values => handleSubmit(values)}
```

Comment:
Use shorthand whenever possible.
```suggestion
      onSubmit={handleSubmit}
```